### DISCUSSION TITLE
Several caches

### DISCUSSION THREAD

**Discussion by tompalmer** [theophanemayaud/video-simili-duplicate-cleaner#120] [Ideas]
Would it be possible to have several caches ? I noticed the cache rebuilds when you change thumbnails options, and it would be good to be able to switch without restarting from scratch

**Comment by theophanemayaud** [comment]
I don’t plan on adding this, but you can already do this if you manually select two different locations for cache depending on what mode you want. 
In fact thinking about it… I’ll check but cached thumbnails are stored regardless of thumbnail options (you do mean compare modes, like cut ends, 2x2… ?). So once you’ve run through both once, it should be able to find cached matches as long as you don’t reset the cache between runs.

**Reply by tompalmer** [reply]
Oh I did not think, ok.
